app/assets/javascripts/components/common/feedback.jsx

```diff
@@ -37,16 +44,32 @@ const Feedback = React.createClass({
     this.hide();
   },
 
+  handleSubmit(event) {
+    const subject = `/revision_feedback?title=${this.titleParam()}`;
+    const body = event.target.value;
+    this.setState({ feedbackSent: 'sending' });
+    API.postFeedbackFormResponse(subject, body).then(() => {
+      this.setState({ feedbackSent: true });
+    });
+    event.preventDefault();
+  },
+
```

Comment:
In the feedback form controller, `create` redirects to the 'confirmation' page. What you can probably do is add a json version of the `confirmation` view and change the url in api.js to add '.json', and then you'll get that back instead of html.

The controller flow is such that there's no explicit error handling, but it will return an error message and code if something on the rails side throws an error. That should be enough to fail the promise (which you could handle with a `.catch()` after the `.then()`).